client/components/data/query-reader-follows/index.jsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { requestFollows } from 'state/reader/follows/actions';
+
+class QueryReaderFollows extends Component {
```

Comment:
So I 100% agree with the sentiment that our QueryComponents are too much boilerplate for accomplishing too little.  And in JS, more than in other langs, lots of boilerplate is really bad because of how it needlessly adds to our bundle size.  I even opened up a PR on this exact topic awhile ago: https://github.com/Automattic/wp-calypso/pull/10861.

I'd be more than happy to brainstorm and come up with a solution to the issue, but I think this feature work isn't the place for that.